src/sync/impl/sync_metadata.cpp

```diff
@@ -113,6 +122,12 @@ realm::Schema make_schema()
         }},
         {c_sync_current_user_identity, {
             {c_sync_current_user_identity, PropertyType::String}
+        }},
+        {c_sync_app_metadata, {
+            {c_sync_app_metadata_deployment_model, PropertyType::String},
+            {c_sync_app_metadata_location, PropertyType::String},
+            {c_sync_app_metadata_hostname, PropertyType::String},
+            {c_sync_app_metadata_ws_hostname, PropertyType::String}
```

Comment:
don't forget to bump the schema version, I can't remember where it is but I don't see it modified anywhere here

Reply:
Adding new types doesn't require bumping the schema version.

Reply:
ah ok thanks!